cargo-quickbuild/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1.0.57"
-cargo = "0.61.1"
+cargo = {version = "0.61.1", path = "../../cargo" }
```

Comment:
Branch is available at https://github.com/alsuren/cargo/pull/1. Remember to point to the public git branch before merging to main.